### DISCUSSION TITLE
Does the Home assistant Frigate add-on use a TMPFS cache?

### DISCUSSION THREAD

**Discussion by LaurenceGough** [blakeblackshear/frigate#3709] [Ask A Question] [answered]
Hi all,

I'm having to build my Pi setup again from scratch and I would like to go back to running Frigate as an add-on in home assistant. I'm probably going to run Home Assistant supervised from a docker container, sort of double supervised.

I'd also like to avoid SSD wear and premature failure by using a TMPFS cache of around 1GB as the standalone Frigate can do. 

Is this possible to configure with the Frigate Add-on?

Many thanks in advance

**Comment by NickM-27** [comment, marked as answer]
[Frigate add-on uses tmpfs cache](https://github.com/blakeblackshear/frigate-hass-addons/blob/main/frigate/config.yaml#L33) however I don't believe the size is customizable (addon limitation)

**Reply by NickM-27** [reply]
To add to this answer, you can see that being used here: https://github.com/blakeblackshear/frigate-hass-addons/blob/main/frigate/config.yaml#L33

**Reply by LaurenceGough** [reply]
Thanks Nick! I found some more info looks like it piggy backs off the tmp volume provided by home assistant. I'll have to do some research to see what the size is but it's good to know my SSD (that didn't auto correct well just then.... Haha) won't wear out too soon.
